src/System.Private.ServiceModel/tests/Common/Infrastructure/tests/project.json

```diff
@@ -5,11 +5,21 @@
     "System.ServiceModel.Primitives": "4.1.0-rc3-23931-00",
     "System.ServiceModel.Security": "4.0.1-rc3-23931-00",
     "xunit": "2.1.0",
-    "xunit.netcore.extensions": "1.0.0-prerelease-00231-07"
+    "xunit.netcore.extensions": "1.0.0-prerelease-00231-07",
+    "test-runtime": {
+      "target": "project",
+      "exclude": "compile"
+    }
   },
   "frameworks": {
     "dnxcore50": {
       "imports": "portable-net45+win8"
     }
+  },
+  "runtimes": {
+    "win7-x64": {},
+    "win7-x86": {},
+    "ubuntu.14.04-x64": {},
+    "osx.10.10-x64": {}
   }
```

Comment:
We are testing on centos, rhel and debian nightly, so I think it's appropriate to add them.  @mellinoe is that something you can add to this PR?